pkg/apis/operator/v1alpha1/common.go

```diff
@@ -99,6 +104,10 @@ type CommonSpec struct {
 	// Override containers' resource requirements
 	// +optional
 	Resources []ResourceRequirementsOverride `json:"resources,omitempty"`
+
+	// Override containers' resource requirements
+	// +optional
+	Version string `json:"version,omitempty"`
```

Comment:
Doesn't this make it part of the CRD effectively? Or are we dropping unknown keys via the OpenAPI schema?

Reply:
You're referring to the member or the json tag? I thought if it wasn't explicitly defined in the schema of the CRD yaml, it wasn't a part of the CRD. I don't know, tbh.

Reply:
Depends on how the CRDs openAPI schema is specified. If it doesn't have a "preserve-unknown-keys" directive then yes, keys we don't specify there will just be dropped (which is what we want in this case).

Reply:
What are you suggesting I do exactly?